compiler/util/traverse_ir.py

```diff
@@ -281,3 +281,31 @@ def do_something(expression, field_name=None):
       all_incidental_actions.setdefault(key, []).extend(incidental_action)
   _fast_traverse_proto_top_down(ir, all_incidental_actions, pattern,
                                 skip_descendants_of, action, parameters or {})
+
+
+def fast_traverse_node_top_down(node, pattern, action, incidental_actions=None,
+                                skip_descendants_of=(), parameters=None):
+  """Traverse a subtree of an IR, executing the given actions.
+
+  fast_traverse_node_top_down is like fast_traverse_ir_top_down, except that:
+
+  It may be called on a subtree, instead of the top of the IR.
+
+  It does not have any built-in incidental actions.
+
+  Arguments:
+    ir: An ir_pb2.Ir object to walk.
```

Comment:
Should be `node`. Same in skip_descendants_of description.

Reply:
Fixed.  Apparently I am far too used to leaning on internal presubmits.